io.github.go_musicfox.go_musicfox.yml

```diff
@@ -0,0 +1,35 @@
+id: io.github.go_musicfox.go_musicfox
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: go-musicfox
+
+finish-args:
+    - --share=network
+    - --socket=pulseaudio
+    - --talk-name=org.freedesktop.Notifications
+
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: go-musicfox
+    buildsystem: simple
+    build-commands:
+      - $GOROOT/bin/go build -trimpath -o go-musicfox cmd/musicfox.go
+      - install -Dm00755 go-musicfox ${FLATPAK_DEST}/bin/go-musicfox
+      - install -Dm00644 app-icon.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm00644 io.github.go_musicfox.go_musicfox.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: git
+        url: "https://github.com/go-musicfox/go-musicfox.git"
+        tag: v4.5.7
+        commit: 1629dd8dbaa2f433bd9011db58ba3c639a15fd0c
+      - type: file
+        path: app-icon.png
+      - type: file
+        path: io.github.go_musicfox.go_musicfox.desktop
```

Comment:
these should be upstream see the note in https://docs.flathub.org/docs/for-app-authors/requirements#required-metadata

Reply:
The upstream icon (1024x1024 png file) is larger than flatpak's limit (512x512), while the upstream dot desktop file works but has a few minor issues. I can try work with the maintainer to (1) include a smaller version of the logo upstream and (2) figure out changes/workarounds needed for the desktop file, thought these changes will take some time to be included in the the next release, depending on what the maintainer thinks. Does that sound like a reasonable solution? I also thought about shrinking the logo with `ImageMagick` in the build process, but that seems unnecessarily complicated.

Reply:
it's not flatpak limit. It's the standard. we just follow standards here. Just delete it in the post-install.

Reply:
oh you mean there is only that?

Reply:
`ffmpeg -i inout.png -vf scale=${size}:${size} output.png`

Reply:
Thank you! Looks like the icon is all good now. I will try to resolve issues with the desktop file next.

Reply:
There is a minor problem with the icon file name in the upstream dot desktop file, so I used `sed` to manually set the icon. That was the only workaround required to switch to the upstream version. All issues mentioned here should be resolved now.

Reply:
don't use sed. Use `desktop-file-edit`

Reply:
Thanks! I just fixed it and pushed the changes.